client/src/components/_common/CustomMessage/CustomMessage.jsx

```diff
@@ -20,14 +20,12 @@ import styles from './CustomMessage.module.scss';
 function CustomMessage({ messageComponentName }) {
   const dispatch = useDispatch();
   const messages = useSelector((state) => {
-    return state.customMessages
-      ? state.customMessages.messages.filter((message) => {
-          return (
-            message.unread &&
-            message.template.component === messageComponentName
-          );
-        })
-      : [];
+    const all = state.customMessages?.messages || [];
```

Comment:
Kind of a nitpick, but probably having a const called `all` could be a little confusing or dicey with the codebase. Maybe allMessages?

Reply:
I opted to just remove that line entirely.